### PR TITLE
feat: descriptive page titles using React Helmet

### PR DESCRIPTION
**Pull request by sigmachirality**
title

### CODE REVIEW

File: src/about.js [hunk 1]
```diff
@@ -144,6 +152,7 @@ const { formatString, ...strings } = LocalizedStrings({
         thanks: `Dziękujemy za używanie Tomato.gg!`,
     },
     ru: {
+        title: "About",
```
Comment (ElijahSink) [inline src/about.js:155]: Could we get this in Russian?
- What was your methodology for getting the other strings that you added?
- What do you think about just using `strings.h1` here?

File: src/search.js [hunk 2]
```diff
@@ -50,6 +51,9 @@ export default withRouter(function Search(props) {
 
     return (
         <div>
+            <Helmet>
+                <title>Tomato.gg</title>
```
Comment (ElijahSink) [inline src/search.js:55]: The sidebar calls this page "Home" so perhaps we could make this `Home - Tomato.gg`? I'm not sure if that's better though.

**Review by ElijahSink — COMMENTED** [review]
I made a couple comments about some minor things. Hopefully I can merge this today (apparently there are some conflicts) unless Goldflag gets to it first.

**Comment by goldflag** [comment]
I probably should've merged earlier today because I made like 10 conflicts lol

**Pull request merged**